content/performance/use-aot-compilation.md

```diff
@@ -6,6 +6,8 @@ author:
   url: https://twitter.com/mgechev
 ---
 
+**Note**: Starting from Angular v9 AOT compiler is the default one, so if you're on Angular v9+ you need to do nothing.
```

Comment:
```suggestion
**Note**: Starting from Angular v9, the AOT compiler is the default compiler, so if you're using Angular v9+ you don't need to take an actions.
```

Reply:
Sounds better with you wording, I would only change the last part for one of the following options:

1) ... don't need to take any actions.
2) ...don't need to take an action.